api/app/controllers/spree/api/v2/storefront/cart_controller.rb

```diff
@@ -14,7 +14,7 @@ def add_item
             variant = Spree::Variant.find(params[:variant_id])
 
             if spree_current_order.nil?
-              render json: { error: "Order doesn't exist" }, status: 422
```

Comment:
@WitekL just raise `ActiveRecord::RecordNotFound` here and BaseController will handle the rest